pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -228,7 +228,7 @@ func checkMachineDriverLibvirtInstalled() (bool, error) {
 	// Check the version of driver if it matches to supported one
 	stdOut, stdErr, err := crcos.RunWithDefaultLocale(libvirtDriverPath, "version")
 	if err != nil {
-		return false, err
+		return false, fmt.Errorf("%v : %s", err, stdErr)
```

Comment:
What will the new error look like? 
I guess something like `FATA exit status 1: /lib64/libvirt.so.0: version `LIBVIRT_3.4.0' not found`?
In the future (not in this PR), we can probably improve the error messages we show to the user, with a user readable error message ("libvirt machine driver is non-functional, maybe crc setup will fix it"), and a more detailed/technical error message, which maybe we will only show on log_level debug ("/lib64/libvirt.so.0: version `LIBVIRT_3.4.0' not found`")